src/librustc_privacy/lib.rs

```diff
@@ -271,7 +271,8 @@ fn def_id_visibility<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>, def_id: DefId)
                     return (ctor_vis, span, descr);
                 }
                 Node::Expr(expr) => {
-                    return (ty::Visibility::Restricted(tcx.hir().get_module_parent(expr.id)),
+                    return (ty::Visibility::Restricted(
+                        tcx.hir().get_module_parent_by_hir_id(expr.hir_id)),
```

Comment:
This formatting doesn't quite make sense to me =P

Reply:
I think I was just trying to squeeze it so that tidy doesn't complain; I can adjust it if need be :P.